proto/src/determined/agent/v1/agent.proto

```diff
@@ -36,6 +78,30 @@ message Agent {
   // The name of the resource pools the agent is in. Only slurm can contain
   // multiples.
   repeated string resource_pools = 6;
+
+  // The slot stats for this agent.
+  SlotStats slot_stats = 11;
```

Comment:
https://github.com/determined-ai/determined/pull/9056#discussion_r1540115545

this should be in required field?